src/utils/Terminal.php

```diff
@@ -37,8 +37,6 @@ abstract class Terminal{
 	public static string $FORMAT_BOLD = "";
 	public static string $FORMAT_OBFUSCATED = "";
 	public static string $FORMAT_ITALIC = "";
-	public static string $FORMAT_UNDERLINE = "";
-	public static string $FORMAT_STRIKETHROUGH = "";
```

Comment:
This is BC breaking and requires a major version.

Reply:
In addition, consoles may have a way to represent these codes, regardless of whether they are supported by Minecraft.